smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftDependency.kt

```diff
@@ -27,6 +27,22 @@ enum class SwiftDependency(
         Resources.computeAbsolutePath("smithy-swift/Packages", "Packages", "SMITHY_SWIFT_CI_DIR") + "/Packages",
         "ClientRuntime"
     ),
+    JSON_RUNTIME(
+        "JSONRuntime",
+        "main",
+        "0.1.0",
```

Comment:
CLIENT_RUNTIME, JSON_RUNTIME and XML_RUNTIME will be versioned the same --- seems like it would make sense to create a constant private string ?